libs/langchain/langchain/document_loaders/parsers/pdf.py

```diff
@@ -376,6 +376,48 @@ class AmazonTextractPDFParser(BaseBlobParser):
     """Send `PDF` files to `Amazon Textract` and parse them.
 
     For parsing multi-page PDFs, they have to reside on S3.
+
+    The AmazonTextractPDFLoader calls the
+    [Amazon Textract Service](https://aws.amazon.com/textract/)
+    to convert PDFs into a Document structure.
+    The loader does pure OCR at the moment, with more features like
+    layout support planned, depending on demand.
```

Comment:
Are we providing layout support here, so this should be corrected.

Reply:
fixed